EXAMPLES.md

```diff
@@ -763,3 +763,44 @@ mix.extract(['chart.js']);
 | grid | Whether to show the grid or not | no | |
 | tooltips | Whether to show tooltips or not | no | |
 | theme | The theme to apply. (an array with 'name' and 'mode'. E.g. ['name' => 'grey', 'mode' => 'light']) | no | |
+
+
+### Tabs
+
+Add the following to the `app.js` file:
+
+```js
+import "@ui/js/tabs.js";
+```
```

Comment:
add this to the readme as that contains the setup for the other components that require specific actions too

Reply:
@ItsANameToo Do I have to add this on README.md or add a notes in the EXAMPLES.md (see below)?

```
> Be sure to follow the instruction on [README.md](/README.md#installation)
```

Reply:
Add the setup to the readme with a quick example on how it should be used (see how it's done for other components). Then the full example and properties overview can be in the example file